src/transforms/onehot.jl

```diff
@@ -65,7 +74,7 @@ function revertfeat(::OneHot, newfeat, fcache)
   
   oname, inds, levels, ordered = fcache
   x = map(zip(columns[inds]...)) do row
-    levels[findfirst(row)]
+    levels[findfirst(==(true), row)]
```

Comment:
@juliohm, this code looks redundant, but it's necessary when `categorical=true`.
See:
```julia
julia> using CategoricalArrays

julia> x = Bool[0, 0, 0, 1, 1, 0]
6-element Vector{Bool}:
 0
 0
 0
 1
 1
 0

julia> findfirst(x)
4

julia> findfirst(categorical(x))
ERROR: TypeError: non-boolean (CategoricalValue{Bool, UInt32}) used in boolean context
...